opentracing-util/src/main/java/io/opentracing/util/GlobalTracer.java

```diff
@@ -131,6 +131,35 @@ public static synchronized boolean registerIfAbsent(final Callable<Tracer> provi
         return false;
     }
 
+    /**
+     * Register a {@link Tracer} to back the behaviour of the {@link #get()}.
+     * <p>
+     * Registration is a one-time operation. Once a tracer has been registered, all attempts at re-registering
+     * will return {@code false}. See {@link #registerIfAbsent(Callable)} for use cases when tracer
```

Comment:
> See `registerIfAbsent(Callable)` for use cases when tracer can be instantiated multiple times.

This contradicts other statements and those use cases are not described where you say they are